src/common/commandExecutor.ts

```diff
@@ -26,6 +26,7 @@ export enum CommandStatus {
 export class CommandExecutor {
     private static ReactNativeCommand = "react-native";
     private currentWorkingDirectory: string;
+    private childProcess = new Node.ChildProcess();
```

Comment:
At the moment we don't need to initialize it on the constructor, although that will probably change when we add more tests to CommandExecutor, and then we'll change this to the constructor.